### DISCUSSION TITLE
Stat panel wasted space

### DISCUSSION THREAD

**Discussion by morphias2004** [grafana/grafana#47288] [Dashboards, Panels & Visualizations]
**What happened**:

Unnecessary amount of wasted space on single stat when graph is turned off.

![image](https://user-images.githubusercontent.com/68760693/161349921-463dc6ae-3ff7-413f-91a0-660355bc2192.png)

Issue is made worse if Query Options are set for Relative Time and/or Time Shift.

![image](https://user-images.githubusercontent.com/68760693/161349971-db382ac4-5c27-4286-a902-2b9986bba64d.png)

Even though these options go into the existing title bar, they rearrange the entire panel.

Making large reduction in the Title and Value text sizes has no effect.

![image](https://user-images.githubusercontent.com/68760693/161350069-c9180fc1-4090-4169-869b-476408a5166c.png)

**What you expected to happen**:

Less wasted space under and to the left of the displayed value.

Adding Query Options should have no effect on the panel layout/size if the data being displayed is essentially the same.

**How to reproduce it (as minimally and precisely as possible)**:

Create a Stat panel with the graph turned off, set the test sizes for Title and Value. I started out with 30 for the Title and 70 for the Value.

Drag the panel to a size just big enough to get the Title and Value to display on separate lines.

That will product a panel with a lot of wasted space to the left and bottom.

Now add a Relative Time and/or a Time Shift to the Query Options. The panel will completely rearrange the layout and necessitate have to make the panel significantly larger, with smaller text sizes, to get the Title and Value to display on two lines again.

![image](https://user-images.githubusercontent.com/68760693/161350105-2d556c65-2437-47dd-92f2-912606018390.png)

**Environment**:
- Grafana version:  8.4.4 "fcfb01faeb"
- Data source type & version:  InfluxDB 2.1.1
- OS Grafana is installed on: Docker
- User OS & Browser: Windows10 x64 with Firefox x64 98.0.2
- Grafana plugins: None
- Others: None

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Comment by morphias2004** [comment]
"Due to the age and number of responses to this discussion, we are deciding to close it."

There has been no response or discussion.  :(
